### ISSUE TITLE
<C-c> and <C-[> don't cancel search

### ISSUE THREAD

**Issue by terrybrashaw** [VSCodeVim/Vim#3668]
**Expected behavior**
Hitting `<C-c>` or `<C-[>` in search mode should exit the search and return to normal mode.

**To Reproduce**
1. Start searching with `/`
2. Type some characters (optional)
3. Hit `<C-c>` or `<C-[>`
4. Search hasn't been canceled.

**Environment:**
Extension version: 1.4.0
VS Code version: Code 1.33.0 (0dd516dd412d42323fc3464531b1c715d51c4c1a, 2019-04-04T15:14:28.026Z)
OS version: Windows_NT x64 10.0.17763

**Comment by jwmann** [comment]
I also have this problem as well but on Mac OS.
It's quite frustrating since `<Esc>` is just another way of doing `<C-[>` not the other way around. 😉

**Comment by kevinsqi** [comment]
Yup, this has been quite frustrating because that's how it worked in vim, and hard to break that muscle memory.

Adding this to settings.json does not appear to work:

```json
  "vim.normalModeKeyBindingsNonRecursive": [
    {
      "before": ["<C-c>"],
      "after": ["<Esc>"]
    }
  ],
```

Any suggestions on how to deal with this?

**Comment by hkleynhans** [comment]
Just tested and it is fixed for me in version 1.8.0.  I'm happy to have this closed.

**Comment by kevinsqi** [comment]
`C-c` works great for me now - thanks for turning this around!
